apps/api/src/app/workflows-v2/usecases/upsert-workflow/upsert-workflow.usecase.ts

```diff
@@ -152,6 +154,11 @@ export class UpsertWorkflowUseCase {
     return command.workflowDto.preferences.user;
   }
 
+  /**
+   * Upsert workflow preferences. While this operation is not typically needed
+   * in a standard workflow update, it's maintained here to support environment
+   * sync scenarios and to enable code reusability.
+   */
```

Comment:
not relevant to this pr:
that is a good comment to have, as it is not straightforward logic.

Reply:
I think I conflicted with this - we should be able to remove this comment as Workflow Pref mutations are now handled in `{Create,Update}Workflow` use-cases now, and I have commented similarly.

Reply:
Conflicts are not a new thing for me ;) 
No Problem if we have it on update/create i will remove this one.